raiden/tests/unit/transfer/mediated_transfer/test_initiator.py

```diff
@@ -1,585 +0,0 @@
-# -*- coding: utf8 -*-
```

Comment:
Why do the test_initiator/mediator/target test files go away?

Reply:
I added a equivalent version for all of them on a previous PR:

[test_initiatorstate.py)](https://github.com/raiden-network/raiden/blob/e760d0fcc662aaa959aeb0084c0921234f473357/raiden/tests/unit/transfer/mediated_transfer/test_initiatorstate.py)
[test_mediatorstate.py](https://github.com/raiden-network/raiden/blob/e760d0fcc662aaa959aeb0084c0921234f473357/raiden/tests/unit/transfer/mediated_transfer/test_mediatorstate.py)
[test_targetstate.py](https://github.com/raiden-network/raiden/blob/e760d0fcc662aaa959aeb0084c0921234f473357/raiden/tests/unit/transfer/mediated_transfer/test_targetstate.py)

Basically the tests that end on `state` are for the version that uses the ChannelState. Since this PR removes the code that doesnt work with ChannelState their companion tests can go away too.